### ISSUE TITLE
All links and radio buttons should be highlighted when focused

### ISSUE THREAD

**Issue by phacks** [theodo/falco#146]
Thanks @AlbericTrancart for the report!

From https://a11yproject.com/checklist/#section-keyboard:

> Can a person navigating with a keyboard, switch, or screen reader see where they currently are on the page?

Currently none of the radio buttons are highlighted, and a few links are not (e.g. “add a metric”)

**Comment by phacks** [comment]
@all-contributors please add @MathildeDuboille for a11y!
